src/uct/ib/base/ib_iface.h

```diff
@@ -148,6 +148,7 @@ struct uct_ib_iface {
         uint8_t             sl;
         uint8_t             traffic_class;
         int                 enable_res_domain;   /* Disable multiple resource domains */
+        size_t              gid_index;           /* IB GID index to use  */
```

Comment:
should be uint8_t and move it after traffic_class to avoid memory waste on padding